source/upgrade/upgrading-mattermost-server.rst

```diff
@@ -12,6 +12,8 @@ Upgrading to the Latest Version
 
 If you are upgrading from version 3.0 or later, these instructions apply to you. If you are upgrading from a version prior to 3.0.0, you must first upgrade to version 3.0.3.
 
+We only support one minor version difference between the server versions when performing a rolling upgrade (for example v5.27.1 + v5.27.2 or v5.26.4 + v5.27.1 is supported, whereas v5.25.5 + v5.27.0 is not supported). Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```

Comment:
@wiersgallak - This is the information we need to confirm as correct or update if needed. Is this still true? A customer has asked for clarification: "it's not clear for me whether I can upgrade from the 5.31 ESR to the 5.37 ESR, skipping everything in between or not.  The note at the end mentions minor versions and rolling releases - as opposed to what?"

Reply:
@wiersgallak I've clarified all upgrade path details under "Upgrading to the Latest Version".